daprdocs/layouts/partials/components/description.html

```diff
@@ -1,15 +1,29 @@
-<p>Table captions:</p>
-<blockquote>
-    <p><code>Status</code>: <a href="/operations/components/certification-lifecycle/">component certification</a> status</p>
-</blockquote>
-<ul>
-    <li><a href="/operations/components/certification-lifecycle/#alpha">Alpha</a></li>
-    <li><a href="/operations/components/certification-lifecycle/#beta">Beta</a></li>
-    <li><a href="/operations/components/certification-lifecycle/#stable">Stable</a></li>
-</ul>
-<blockquote>
-    <p><code>Since</code>: the version of the Dapr Runtime in which the component first moved to the current status</p>
-</blockquote>
-<blockquote>
-    <p><code>Component version</code>: the version of the component</p>
-</blockquote>
+<p>Table headers to note:</p>
+
+<table>
+    <tr>
+      <th>Header</th>
+      <th>Description</th>
+      <th>Example</th>
+    </tr>
+    <tr>
+      <td>Status</td>
+      <td><a href="/operations/components/certification-lifecycle/">Component certification</a> status</td>
+      <td>
+        <a href="/operations/components/certification-lifecycle/#alpha">Alpha</a><br>
+        <a href="/operations/components/certification-lifecycle/#beta">Beta</a><br>
+        <a href="/operations/components/certification-lifecycle/#stable">Stable</a><br>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>Component version</td>
+      <td>The version of the component</td>
+      <td>v1</td>
+    </tr>
+    <tr>
+      <td>Since runtime version</td>
+      <td>The version of the Dapr Runtime in which the component <br> first moved to the current status</td>
+      <td>1.11</td>
```

Comment:
What is this 1.11 here for?

Reply:
the existing table has an 'example' column, so 1.11 is the example of the runtime version that could be seen below